Core/Core/AppEnvironment/FeatureFlags/GetEnvironmentFeatureFlags.swift

```diff
@@ -0,0 +1,76 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2022-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import CoreData
+import Foundation
+
+public enum EnvironmentFeatureFlags: String {
+    case send_usage_metrics
+}
+
+public class GetEnvironmentFeatureFlags: APIUseCase {
+    public typealias Model = FeatureFlag
+    public let context: Context
+
+    public var scope: Scope {
+        let context = NSPredicate(format: "%K == %@", #keyPath(FeatureFlag.canvasContextID), self.context.canvasContextID)
+        let enabled = NSPredicate(format: "%K == true", #keyPath(FeatureFlag.enabled))
```

Comment:
I wonder if filtering only to enabled flags makes sense. If we go this way then saving disabled flags isn't necessary because they won't be read back from the DB, in that case we could just remove them from CoreData.